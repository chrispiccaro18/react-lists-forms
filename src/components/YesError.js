export default function YesError() {
  throw 'Uh-oh!';
}
