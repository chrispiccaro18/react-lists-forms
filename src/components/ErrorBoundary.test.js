import React from 'react';
import { shallow } from 'enzyme';
import ErrorBoundary from './ErrorBoundary';
import YesError from './YesError';

describe('ErrorBoundary component', () => {
  it('renders ErrorBoundary component', () => {
    const wrapper = shallow(
      <ErrorBoundary>
        <YesError />
      </ErrorBoundary>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
