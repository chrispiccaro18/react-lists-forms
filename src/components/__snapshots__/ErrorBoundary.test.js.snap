// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary component renders ErrorBoundary component 1`] = `<YesError />`;
